### DISCUSSION TITLE
CVE-2024-1597 Vulnerability  in quarkus-jdbc-postgresql dependency

### DISCUSSION THREAD

**Discussion by ruhilrinku** [quarkusio/quarkus#38998] [Community]
CVE-2024-1597 Vulnerability  in quarkus-jdbc-postgresql dependency. 

ETA please ?

**Comment by gsmet** [comment]
The update has been merged to the `main` branch and will be released as part of `3.8.1` on March 6th.

Note that as explained in the CVE description you pointed, people using the default query mode are NOT affected.

If you think you may be affected, you can enforce the driver version in your local POM files while waiting for March 6th.

**Reply by ruhilrinku** [reply]
Thanks for quick response. It is not allowing me to enforce the newer driver version with gradle. an example will help. Thanks in advance

**Reply by gsmet** [reply]
In the end, it will be included in this Wednesday's release.

**Reply by gsmet** [reply]
3.8.1 was released today and it contains the update.
